### PR TITLE
Add error messages, tests, and delete duplicate property files

### PR DESCRIPTION
**Pull request by brenthdaniel**
PR for #7563

### CODE REVIEW

**Comment by brenthdaniel** [comment]
#build

**Comment by brenthdaniel** [comment]
random unrelated failure
#build again

File: dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops [hunk 1]
```diff
@@ -756,6 +756,8 @@ info.resumeFailedException=Resuming the {0} server failed. Check the server logs
 info.resumingListeners.target=Resuming the specified target components of the {0} server.
 info.resumedListeners.target=Resuming the specified target components of the {0} server completed.
 info.resumeFailedException.target=Resuming the specified target components of the {0} server failed. Check the server logs for details.
+info.serverCommandAuthFailure=The command {0} failed because the process could not delete the file {1}.
```
Comment (Charlotte-Holt) [inline dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Change
"The command {0} failed because the process **could** not delete the file {1}."
to
"The command {0} failed because the process **might** not delete the file {1}."

Comment (brenthdaniel) [reply dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Hi, 
I'm not sure that change makes sense here. If this error message is issued, it is definite that the file was not able to be deleted. 'Might' would imply that it may or may not have been able to delete the file. It would also be inconsistent with other failures from the same script, all of which use 'could not' when something was not able to be completed.

Comment (Charlotte-Holt) [reply dev/com.ibm.ws.kernel.boot/resources/com/ibm/ws/kernel/boot/resources/LauncherMessages.nlsprops]: Hi @brenthdaniel. I understand, and thanks for your feedback. Okay to leave it as is.
